src/commands.cpp

```diff
@@ -3634,6 +3604,24 @@ void CommandGetUserQuota::procresult()
                 break;
 
             case EOO:
+                if (got_storage)
+                {
+                    if (details->storage_used > details->storage_max)
```

Comment:
>= ??

Reply:
Updated to use the most conservative options to prevent possible problems